### PR TITLE
Fix bug in Script.include when urls is empty

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: libraries/script-engine/src/BatchLoader.cpp [hunk 1]
```diff
@@ -38,6 +38,12 @@ void BatchLoader::start() {
 
     _started = true;
 
+    if (_urls.size() == 0) {
+        _finished = true;
+        emit finished(_data);
+        return;
+    }
```
Comment (huffman) [inline libraries/script-engine/src/BatchLoader.cpp]: This doesn't actually get used now, but it fixes a problem with it. I tested it before adding the short-circuit in ScriptEngine.

**Comment by huffman** [comment]
Test:
 * Confirm defaultScripts.js is working
 * Restarting defaultScripts.js is working
 * Stopping defaultScripts.js is working

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a0266edb1a520b47e0160a86bf4bf4bce6de19 available at: https://gist.github.com/57a1ebb9026d85041ba8890f9f4f5835<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8842-73a0266.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8842-73a0266.dmg

**Comment by ZappoMan** [comment]
Quick test:
- close default scripts - PASS
- lasers work - PASS

**Pull request merged**